src/transformers/models/wav2vec2/modeling_wav2vec2.py

```diff
@@ -1611,7 +1611,6 @@ def __init__(self, config):
         self.post_init()
 
     @add_start_docstrings_to_model_forward(WAV_2_VEC_2_INPUTS_DOCSTRING)
-    @replace_return_docstrings(output_type=Wav2Vec2BaseModelOutput, config_class=_CONFIG_FOR_DOC)
```

Comment:
This class is deprecated and we don't document it anymore. So removing the docstring here completely as it was failing